src/mongo.ts

```diff
@@ -348,6 +348,26 @@ export class Mongo implements DBInterface {
     );
   }
 
+  public async unsetService(
+    userId: string,
+    serviceName: string
+  ): Promise<void> {
+    const id = this.options.convertUserIdToMongoObjectId
+      ? toMongoID(userId)
+      : userId;
+    await this.collection.update(
```

Comment:
use `updateOne`?

Reply:
Would be better but actually all code use update so for consistency maybe let's keep this change for another pr?

Reply:
sure